projects/fsastorefrontlib/src/shared/util/helpers/PaginationHelper.ts

```diff
@@ -0,0 +1,33 @@
+import { PaginationModel } from '@spartacus/core';
+
+export class PaginationHelper {
+  public static getPaginationResults(pagination: PaginationModel, collection) {
+    let paginatedCollection;
+    const totalResults: number = collection.length;
+    let totalPages: number = Math.floor(totalResults / pagination.pageSize);
+    if (totalResults % pagination.pageSize > 0) {
+      totalPages++;
+    }
```

Comment:
Could this have been calculated just with `Math.round(totalResults / pagination.pageSize)`?

Reply:
No because we need integer number of pages and if we have some remainder page size should be increased. I didn't find the function which will divide two numbers and round integer to the higher number.

Reply:
`Math.ceil(totalResults / pagination.pageSize)` then?

Reply:
Acctualy Math.ceil funciton doing that job :)